engine/src/main/java/org/camunda/bpm/engine/management/JobDefinition.java

```diff
@@ -97,4 +97,10 @@
    */
   String getTenantId();
 
+  /**
+   * The id of the deployment this job definition is related to. In a deployment-aware setup,
+   * this leads to all jobs of the same definition being executed on the same node.
+   */
+  String getDeploymentId();
```

Comment:
Another request I would like to raise is to always expose newly added API getters to the REST API and cover it with REST API reference docs. I think this should be raised in the team meeting as well to discuss whether we would like to make it a general rule.

Reply:
In general I totally agree, but I guess it also depends on the task itself in which this new API is created. If, as in this case here, it is simply a means to make progress on a different topic than adding the getter itself, I would rather vote for creating a follow-up ticket to add it to the REST API (which I haven't done here yet either though).

Reply:
I simply have the concern that this rule otherwise might blow up the original topic too much.

Reply:
I think it could make sense to have a general rule since it's super easy and fast to achieve it for the one implementing and brings much value for users. How a developer achieves it in the end (e. g. with a follow-up ticket) is a means of individual project management and should not be regulated.

Reply:
Totally agree 👍

Reply:
Added it to the DTO

Reply:
Let's add a test & docs to the REST API reference

Reply:
Tests added, documentation will follow 😃

Reply:
Documentation in https://github.com/camunda/camunda-docs-manual/pull/501